sdk/communication/Azure.Communication.Identity/src/CommunicationIdentityClient.cs

```diff
@@ -290,6 +302,57 @@ public virtual async Task<Response> RevokeTokensAsync(CommunicationUserIdentifie
             }
         }
 
+        /// <summary>Issues a token for a <see cref="CommunicationUserIdentifier"/>.</summary>
+        /// <param name="communicationUser">The <see cref="CommunicationUserIdentifier"/> for whom to issue a token.</param>
+        /// <param name="cancellationToken">The cancellation token to use.</param>
+        /// <exception cref="RequestFailedException">The server returned an error.</exception>
+        public virtual Response<CommunicationTurnCredentialsResponse> IssueTurnCredentials(CommunicationUserIdentifier communicationUser, CancellationToken cancellationToken = default)
+        {
+            var RestClientForTurn = new CommunicationIdentityRestClient(
+                _clientDiagnostics,
+                _pipeline,
+                _endpoint,
+                apiVersion: "2021-12-12");
```

Comment:
is not turn on a preview version?

Reply:
Not yet. The change to move the TURN service endpoint to a preview version was checked-in yesterday (it's in Dev/Int, but not in PROD yet). So, I need to coordinate this based on when the service change will get deployed to PROD.

Reply:
This will also be updated after the service with preview version is deployed